force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,12 +314,21 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
-
+        
+        Integer theSum = 0;
+
+        for(Integer num : numbers)
```

Comment:
Be aware of the formatting here. If you want to hang the brackets on the next line or keep them on the previous line, that is fine. Both are acceptable. Just dont switch back and forth.